integration/secrets/secret_test.go

```diff
@@ -139,14 +139,14 @@ func TestEncryptionEndToEnd(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	keyBytes, err := base64.RawURLEncoding.DecodeString(info.Spec.PublicKeys[0].KeyID)
+	keyBytes, err := base64.RawURLEncoding.DecodeString(info[0].Spec.PublicKeys[0].KeyID)
```

Comment:
will this always have something at info[0]?

Reply:
Either something at [0], or nothing, depending on hub implemention. Right now it always has [0]